tests/QS/regtest-gw/TEST_FILES

```diff
@@ -1,11 +1,11 @@
-G0W0_H2O_PBE0.inp                                     11      1e-09            -17.092142638413101
-evGW_H2O_PBE.inp                                      11      1e-08            -17.052375554380653
+G0W0_H2O_PBE0.inp                                     11      1e-09            -17.092199812564395
+evGW_H2O_PBE.inp                                      11      1e-08            -17.052437673881482
 evGW_H2O_PBE_Fermi_level_offset.inp                   11      1e-08            -17.102203857645577
-evGW_OH_PBE.inp                                       11      1e-08            -16.414924714798197
-evGW_H2O_PBE_RI_HFX.inp                               11      1e-08            -13.112415607677665
-G0W0_H2O_PBE_periodic.inp                             11      1e-07            -17.143194694023901
+evGW_OH_PBE.inp                                       11      1e-08            -16.414920515710477
+evGW_H2O_PBE_RI_HFX.inp                               11      1e-08            -13.112474185087502
+G0W0_H2O_PBE_periodic.inp                             11      1e-07            -17.144843116457857
 G0W0_H2O_PBE_pade.inp                                 78    1.0E-14                          16.22
-evGW_OH_PBE_svd.inp                                   11      1e-08            -16.414906390703532
-evGW_H2O_PBE_RI_HFX_svd.inp                           11      1e-08            -13.112371113806535
-scGW0_H2O_PBE.inp                                     11      1e-08            -17.118381127539799
+evGW_OH_PBE_svd.inp                                   11      1e-08            -16.452864482288359
+evGW_H2O_PBE_RI_HFX_svd.inp                           11      1e-08            -13.176028670946073
+scGW0_H2O_PBE.inp                                     11      1e-08            -17.118441200805574
```

Comment:
Why does this commit change regtest values?

Reply:
It's because of the new defaults for the integrals: GPW for MP2 in any case, Obara-Saika integrals for non-periodic systems, GPW for periodic, Gamma-only calculations, Obara-Saika for periodic systems with k-points. For non-GW regtests, I have added ERI_METHOD GPW in the input to keep the regtest values, for GW regtests, I kept the input as is with the default ERI_METHOD while updating the reference values.